workspaces/orchestrator/app-config.yaml

```diff
@@ -62,7 +62,23 @@ techdocs:
 
 auth:
   # see https://backstage.io/docs/auth/ to learn about auth providers
-  providers: { guest: {} }
+  environment: development
+  providers:
+    guest: {}
+    github:
+      development:
+        clientId: ${AUTH_GITHUB_CLIENT_ID}
```

Comment:
will this work also if these env vars are not defined?

Reply:
yes, just the authentication via GH fails then. But the app starts.

Reply:
scratch that, I see the login option via github is available and fails anyhow, so this is better